calblueprint/wecaresolar
# Topic Cards

## 🖋 PR Description

Created and style topic cards and implemented card scroll view. Topics in the initial state should be rendered on the guides tab. Clicking on the topic card should render the topic views page.

Notes:
- Image URLs are still expiring Dropbox links. Are we uploading the images and accessing them locally are referencing them from an external source?

### 🤳Screenshots (if applicable)
<img width="495" alt="Screen Shot 2020-12-01 at 8 58 58 PM" src="https://user-images.githubusercontent.com/57604304/100830312-3223d280-3418-11eb-9962-995ab90ba7ab.png">
<img width="497" alt="Screen Shot 2020-12-01 at 8 59 46 PM" src="https://user-images.githubusercontent.com/57604304/100830314-33ed9600-3418-11eb-9d91-85ac85cec3f8.png">


### 🐞 Bug Report

I ran into a bug when attempting to link the topic cards to the topic views because that routes for lessons and topics were overlapping. To differentiate the routes, lesson and topic were added to the paths in guides.tsx.

### 🧪 Testing

How do we test your feature?

NPM start -- The topic cards should be rendered underneath Susan's playlist cards. Click on different topic cards.


cc: @joelenelatief
